hw/ip/lc_ctrl/doc/programmers_guide.md

```diff
@@ -10,11 +10,18 @@ Hence the following programming sequence applies to both SW running on the devic
 3. Claim exclusive access to the transition interface by writing kMuBi8True to the [`CLAIM_TRANSITION_IF`](../data/lc_ctrl.hjson#claim_transition_if) register, and reading it back. If the value read back equals to kMuBi8True, the hardware mutex has successfully been claimed and SW can proceed to step 4. If the value read back equals to 0, the mutex has already been claimed by the other interface (either CSR or TAP), and SW should try claiming the mutex again.
 Note that all transition interface registers are protected by the hardware-governed [`TRANSITION_REGWEN`](../data/lc_ctrl.hjson#transition_regwen) register, which will only be set to 1 if the mutex has been claimed successfully.
 
-4. If required, enable the external clock and other vendor-specific OTP settings in the [`OTP_VENDOR_TEST_CTRL`](../data/lc_ctrl.hjson#otp_vendor_test_ctrl) register.
-Note that these settings only take effect in RAW, TEST* and RMA life cycle states.
-They are ignored in the PROD* and DEV states.
-
-5. Write the desired target state to [`TRANSITION_TARGET`](../data/lc_ctrl.hjson#transition_target). For conditional transitions, the corresponding token has to be written to [`TRANSITION_TOKEN_0`](../data/lc_ctrl.hjson#transition_token_0). For all unconditional transitions, the token registers have to be set to zero.
+4. If required, software can enable optional settings via the the [`OTP_VENDOR_TEST_CTRL`](../data/lc_ctrl.hjson#otp_vendor_test_ctrl) register:
+    - In RAW, TEST* and RMA life cycle states, software can switch to the external clock via the [`OTP_VENDOR_TEST_CTRL.EXT_CLK_EN`](../data/lc_ctrl.hjson#otp_vendor_test_ctrl#ext_clk_en) register.
+    This setting is ignored in ignored in the PROD* and DEV states.
+    - Software can request a volatile RAW -> TEST_UNLOCKED0 transition by setting the [`OTP_VENDOR_TEST_CTRL.VOLATILE_RAW_UNLOCK`](../data/lc_ctrl.hjson#otp_vendor_test_ctrl#volatile_raw_unlock) to 1.
```

Comment:
nit: Does it make sense to put the VOLATILE_RAW_UNLOCK token into its own documentation sub-section to make it clear that it is a test-chip only feature?

Reply:
Do you mean a separate section here in the programmers guide?

I was just trying to avoid duplication because the rest of the sequence is almost identical.
But we can split it out to make explicit that it is intended for test-chips only.

Reply:
I have split this out into a separate section in the docs.

Reply:
Thanks! LG